android/lib/src/main/java/com/ern/api/impl/navigation/ElectrodeNavigationActivityListener.java

```diff
@@ -0,0 +1,39 @@
+package com.ern.api.impl.navigation;
+
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import com.ern.api.impl.core.ElectrodeBaseFragmentDelegate;
+
+import org.json.JSONObject;
+
+public interface ElectrodeNavigationActivityListener extends ElectrodeBaseFragmentDelegate.ElectrodeActivityListener {
+    /**
+     * Use to delegate a navigate call to the parent activity.
+     *
+     * @param pageName {@link String} MiniApp view component name or the next page to be navigated to.
+     * @param data     {@link Bundle} Data associated with this navigation.
+     * @return true | false
+     */
+    boolean navigate(@Nullable String pageName, @NonNull Bundle data);
```

Comment:
What does it mean to "navigate to null"? Is this a valid scenario? If yes, it would be good to explain this in the Javadoc. If no, any chance we can make the first parameter non-null?